service/messageHandler.go

```diff
@@ -26,41 +27,48 @@ var predicates = map[string]string{
 	"http://www.ft.com/ontology/annotation/mentions":           "mentions",
 }
 
+type kafkaProducer interface {
+	SendMessage(message kafka.FTMessage) error
```

Comment:
Only this function is necessary for the definition of the interface. The others can be removed.